pact.cabal

```diff
@@ -292,7 +316,7 @@ executable gasmodel
   default-language:    Haskell2010
 
 test-suite hspec
-  if !impl(ghcjs)
+  if !impl(ghcjs) && flag(build-tool)
     main-is:          hspec.hs
     type:             exitcode-stdio-1.0
   else
```

Comment:
This branch will be selected for `!impl(ghcjs) && !flag(build-tool)`, which probably is not intended.

```cabal
  elif impl(ghcjs) && flag(build-tool)
    main-is:          hspec-ghcjs.hs
    type:             exitcode-stdio-1.0
  else
    buildable: False
```

Reply:
It's intended this way to at least run some tests on ghc && !build-tool. What is really needed is a subset of tests for this case but that can wait.

Reply:
Oh, cool. I though the build-tool would be incompatible with the test suite. Much better this way. We should also reenable tests in the ci build for -build-tool.